demo/src/main/java/com/example/demo/DemoApplicationForSQ.java

```diff
@@ -5,4 +5,11 @@
 	public void doSomething() {
 	}
 
+	public void doSomething2() {
+	}
+
+	public int doSomething3() {
+		return 77771;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS3400)